templates/web/fixmystreet.com/about/open311-api-info.html

```diff
@@ -0,0 +1,154 @@
+[% INCLUDE 'header.html', title = 'Open311', bodyclass = 'twothirdswidthpage' %]
```

Comment:
title here should probably match what you have in the h1 just below

Reply:
Ah yes, I missed that, because initially I was updating the original Open311 page. Thanks.